### ISSUE TITLE
Prism mock command fails with dependancy issue

### ISSUE THREAD

**Issue by harish-ramesh** [stoplightio/prism#1282]
## Describe the bug

Updated the prism npm package today and then tried using the mock command, the command failed with the following error:

```
internal/modules/cjs/loader.js:775
    throw err;
    ^

Error: Cannot find module 'lodash/fp'
Require stack:
- /usr/local/lib/node_modules/@stoplight/prism-cli/node_modules/@stoplight/prism-http/dist/validator/validators/security/index.js
- /usr/local/lib/node_modules/@stoplight/prism-cli/node_modules/@stoplight/prism-http/dist/validator/index.js
- /usr/local/lib/node_modules/@stoplight/prism-cli/node_modules/@stoplight/prism-http/dist/mocker/callback/callbacks.js
- /usr/local/lib/node_modules/@stoplight/prism-cli/node_modules/@stoplight/prism-http/dist/mocker/index.js
- /usr/local/lib/node_modules/@stoplight/prism-cli/node_modules/@stoplight/prism-http/dist/forwarder/index.js
- /usr/local/lib/node_modules/@stoplight/prism-cli/node_modules/@stoplight/prism-http/dist/index.js
- /usr/local/lib/node_modules/@stoplight/prism-cli/dist/util/createServer.js
- /usr/local/lib/node_modules/@stoplight/prism-cli/dist/commands/mock.js
- /usr/local/lib/node_modules/@stoplight/prism-cli/dist/index.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:772:15)
    at Function.Module._load (internal/modules/cjs/loader.js:677:27)
    at Module.require (internal/modules/cjs/loader.js:830:19)
    at require (internal/modules/cjs/helpers.js:68:18)
    at Object.<anonymous> (/usr/local/lib/node_modules/@stoplight/prism-cli/node_modules/@stoplight/prism-http/dist/validator/validators/security/index.js:7:14)
    at Module._compile (internal/modules/cjs/loader.js:936:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:947:10)
    at Module.load (internal/modules/cjs/loader.js:790:32)
    at Function.Module._load (internal/modules/cjs/loader.js:703:12)
    at Module.require (internal/modules/cjs/loader.js:830:19) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/usr/local/lib/node_modules/@stoplight/prism-cli/node_modules/@stoplight/prism-http/dist/validator/validators/security/index.js',
    '/usr/local/lib/node_modules/@stoplight/prism-cli/node_modules/@stoplight/prism-http/dist/validator/index.js',
    '/usr/local/lib/node_modules/@stoplight/prism-cli/node_modules/@stoplight/prism-http/dist/mocker/callback/callbacks.js',
    '/usr/local/lib/node_modules/@stoplight/prism-cli/node_modules/@stoplight/prism-http/dist/mocker/index.js',
    '/usr/local/lib/node_modules/@stoplight/prism-cli/node_modules/@stoplight/prism-http/dist/forwarder/index.js',
    '/usr/local/lib/node_modules/@stoplight/prism-cli/node_modules/@stoplight/prism-http/dist/index.js',
    '/usr/local/lib/node_modules/@stoplight/prism-cli/dist/util/createServer.js',
    '/usr/local/lib/node_modules/@stoplight/prism-cli/dist/commands/mock.js',
    '/usr/local/lib/node_modules/@stoplight/prism-cli/dist/index.js'
  ]
}

```

## To Reproduce

1. Install the latest version by using
```
npm install -g @stoplight/prism-cli
```
2. Tried creating a mock server using a sample swagger specification.

**Comment by Arisa93** [comment]
Same issue for me as well.

**Comment by XVincentX** [comment]
Yes, I have noticed this as well just now that you've pointing this out. I'm checking this out, but I am suspecting a dependency has been changed and that broke something. I'll keep you posted.

**Comment by XVincentX** [comment]
Ok it seems like lodash has just published 4.17.17 that has a broken import (specifically the fp part is missing the index.js file)

**Comment by XVincentX** [comment]
@Arisa93 @harish-ramesh

**Comment by lsheva** [comment]
For those who experiencing issues with nested lodash dependency and use yarn adding to package.json solves the issue:
```  
"resolutions": {
    "lodash": "4.17.15"
}
```

**Comment by XVincentX** [comment]
I'm gonna wait could of hours to see if they can solve this quickly and publish 4.17.18, otherwise I'll release a patch release pinning it to the v15

**Comment by XVincentX** [comment]
I have just published v3.3.6 that pins to lodash 4.17.15 that should solve the issue.

**Comment by mathiasbynens** [comment]
Please give `lodash@4.17.18` a try. Details here: https://github.com/lodash/lodash/issues/4837#issuecomment-655609600
